test/integration_test/basic_test.go

```diff
@@ -107,6 +107,34 @@ var testStorageClusterBasicCases = []types.TestCase{
 		},
 		TestFunc: BasicUpgradeStorageCluster,
 	},
+	{
+		TestName:        "BasicUpgradeStorageClusterWithAllComponents",
```

Comment:
this should cover everything in the basic upgrade test case, if this test passes, we can run this test only in our pipeline, but let's keep the default upgrade case for now